netmiko/extreme/extreme_ssh.py

```diff
@@ -1,13 +1,123 @@
 """Extreme support."""
 from __future__ import unicode_literals
+import re
+import time
 from netmiko.cisco_base_connection import CiscoSSHConnection
 
 
+QUESTIONS = {
+    'save': 'Do you want to save configuration to',
+}
+
+
 class ExtremeSSH(CiscoSSHConnection):
     """Extreme support."""
+
     def session_preparation(self):
-        """Extreme requires enable mode to disable paging."""
+        """
+        Extreme requires enable mode to disable paging.
+        """
         self._test_channel_read()
-        self.enable()
         self.set_base_prompt()
         self.disable_paging(command="disable clipaging\n")
+
+    def answer_questions(self, *args, **kwargs):
+        """
+        Send questions/answers with send_command_timing.
+        """
+        if len(args) >= 1:
+            command_string = args[0]
+
+        # handle commands which return a question
+        output = self.send_command_timing(command_string)
+        if QUESTIONS[command_string] in output:
+            output += self.send_command_timing('yes')
+
+        return output
+
+    def send_command(self, *args, **kwargs):
+        """
+        Catch questions from the CLI.
+        Handle prompt changes during command executions.
+        """
+        if len(args) >= 1:
+            command_string = args[0]
+
+        # command that prompt a question
+        if command_string in QUESTIONS.keys():
+            output = self.answer_questions(command_string)
```

Comment:
I need to make the behavior of `send_command` uniform across a bunch of platforms. So I don't think we should do this.

Instead, I think we might want to just make a `save_config` method (as that is something that is probably worth expanding out to other platforms).